doc/source/merging.rst

```diff
@@ -1098,7 +1098,8 @@ This is equivalent but less verbose and more memory efficient / faster than this
 Joining with two multi-indexes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-This is not Implemented via ``join`` at-the-moment, however it can be done using the following.
+.. versionadded:: 0.21.0
+As of version 0.21.0 joining on two multi-indexes is possible:
```

Comment:
can you add a similar example to a sub-section in whatsnew (and put a ref to here). you can use this example and show how we *could* do it previously (e.g. the below section), and how it will just work now.

Reply:
Shall I wait to include it in 0.21.1?

Reply:
going to be for next version (so you can wait to move it is ok), after we tag can merge shortly after.